thirdparty/runtime_environments_test.go

```diff
@@ -21,3 +22,22 @@ func TestGetImageNames(t *testing.T) {
 	assert.NotEmpty(result)
 	assert.NotContains(result, "")
 }
+
+func TestGetPackages(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetPackages")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	opts := PackageFilterOptions{
+		Page:    0,
+		Limit:   10,
+		Manager: "pip",
+	}
+	result, err := c.getPackages(ctx, "ami-0e12ef25a5f7712a4", opts)
```

Comment:
why did we choose to have AMI passed in here separate from PackageFilterOptions?

Reply:
The AMI is a required argument, whereas the options are optional arguments for the API call.

Reply:
I think this can still be part of opts, even if its required, and then we should just validate that it's non-empty (which is good practice to do anyway if the value is required)

Reply:
Looked at this a bit more, and we aren't actually consistent about this, and there aren't too many inputs here anyway.